sjb/config/test_cases/ami_build_origin_int_rhel_fork.yml

```diff
@@ -0,0 +1,246 @@
+---
+parameters:
+- name: "TRELLO_URL"
+  description: "The resulting ami will be named to reference the specified card and a comment will be added to the corresponding trello card when the ami completes.
+If you don't have a corresponding trello card.  You can use anything that will get added to the ami name => fork_ami_openshift3_${TRELLO_URL}_${BUILD_NUMBER}"
+- name: "ORIGIN_FORK_ID"
+  description: "The fork id to use for the origin repo."
+  default_value: "openshift"
+- name: "ORIGIN_BRANCH"
+  description: "Branch to use for origin fork."
+  default_value: "master"
+- name: "ORIGIN_REPO"
+  description: "The name of the origin repo."
+  default_value: "origin"
+- name: "S2I_FORK_ID"
+  description: ""
+  default_value: "openshift"
+- name: "S2I_BRANCH"
+  description: ""
+  default_value: "master"
+- name: "S2I_REPO"
+  description: "Note: s2i fork amis are not currently supported until we drop golang 1.4 support.  Build an s2i binary manually and give it to test."
+  default_value: "source-to-image"
+- name: "RUN_EXTENDED"
+  description: "In addition to the verification tests include the extended tests.  Note:  The ami will still be marked qe-ready and the qe tests launched even if these tests fail."
+  default_value: "false"
+- name: "RUN_EXTENDED"
+  description: "Run the QE 'smoke' tests after build and basic verification success.  These tests should be run for most amis."
+  default_value: "true"
```

Comment:
I think we only support string variables right now -- should debug to see how "boolean" variables are done by Jenkins, shell vars are always strings anyway. That may be less effort than changing the generator stuff to honor different types of variables.